src/main/java/com/conveyal/datatools/common/status/MonitorableJob.java

```diff
@@ -67,7 +67,8 @@
         VALIDATE_ALL_FEEDS,
         MONITOR_SERVER_STATUS,
         MERGE_FEED_VERSIONS,
-        RECREATE_BUILD_IMAGE
+        RECREATE_BUILD_IMAGE,
+        NOTIFY_USERS_FOR_SUBSCRIPTION_JOB
```

Comment:
Should not be a monitorable job according to reasons listed in https://github.com/ibi-group/datatools-server/pull/348#discussion_r512341045.

Reply:
Updated.